src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -439,6 +460,19 @@ private void updateTaskStateTo(String taskId, TaskState newState) {
         camundaServiceApi.addLocalVariablesToTask(authTokenGenerator.generate(), taskId, camundaLocalVariables);
     }
 
+    private void performCancelTaskAction(String taskId) {
+        Map<String, String> body = new ConcurrentHashMap<>();
+        body.put("escalationCode", ESCALATION_CODE);
+        try {
+            camundaServiceApi.bpmnEscalation(authTokenGenerator.generate(), taskId, body);
+        } catch (FeignException ex) {
```

Comment:
Could we use something like Feign ErrorDecoder or ControllerAdvice to help keep the code cleaner?  Maybe something to think about as a future improvement across all apis?

Reply:
yes we have to use Rest exception handler or ErrorDecoder to handle FeignExceptions. 
We can create a tech debt story for this.

Reply:
Yes, create a tech debt ticket for that please.

Reply:
https://tools.hmcts.net/jira/browse/RWA-282